app/bundles/PageBundle/Tests/Controller/PageControllerTest.php

```diff
@@ -122,4 +125,52 @@ public function LandingPageTrackingSecondVisit()
         $this->assertCount(1, $eventLogsAfterSecondVisit);
         $this->assertSame(reset($eventLogsAfterFirstVisit)['id'], reset($eventLogsAfterSecondVisit)['id']);
     }
+
+    /**
+     * Test tracking of a first visit with UTM Tags.
+     */
+    public function testLandingPageWithUtmTracking(): void
+    {
+        $timestamp  = \time();
+        $page       = $this->createTestPage();
+
+        $this->client->request('GET', "{$page->getAlias()}?utm_source=linkedin&utm_medium=social&utm_campaign=mautic&utm_content=".$timestamp);
+        $clientResponse = $this->client->getResponse();
+        // $this->assertEquals(Response::HTTP_OK, $clientResponse->getStatusCode(), 'page not found');
+        if (Response::HTTP_OK === $clientResponse->getStatusCode()) {
+            $leadModel     = $this->container->get('mautic.model.factory')->getModel('lead');
+            $allUtmTags    = $leadModel->getUtmTagRepository()->getEntities();
+            $this->assertNotCount(0, $allUtmTags);
+
+            foreach ($allUtmTags as $utmTag) {
+                $this->assertSame('linkedin', $utmTag->getUtmSource(), 'utm_source does not match');
+                $this->assertSame('socissal', $utmTag->getUtmMedium(), 'utm_medium does not match');
+                $this->assertSame('mautic', $utmTag->getUtmCampaign(), 'utm_campaign does not match');
+                $this->assertSame(strval($timestamp), $utmTag->getUtmContent(), 'utm_content does not match');
+            }
+        }
+    }
+
+    /**
+     * Create a page for testing.
+     */
+    protected function createTestPage($pageParams=[]): Page
+    {
+        $page = new Page();
+
+        $title       = isset($pageParams['title']) ? $pageParams['title'] : 'Page:Page:LandingPageTracking';
```

Comment:
The PHP 7 syntax can be used for shorter, readable code
```suggestion
        $title       = $pageParams['title'] ?? 'Page:Page:LandingPageTracking';
```
Again, nothing that needs to change. It does the same thing. Just FYI.